src/modules/poweraccent/PowerAccent.Core/Services/SettingsService.cs

```diff
@@ -0,0 +1,180 @@
+                        switch (settings.Properties.ToolbarPosition.Value)
+                        {
+                            case "Top center":
+                                Position = Position.Top;
+                                break;
+                            case "Bottom center":
+                                Position = Position.Bottom;
+                                break;
+                            case "Left":
+                                Position = Position.Left;
+                                break;
+                            case "Right":
+                                Position = Position.Right;
+                                break;
+                            case "Top right corner":
+                                Position = Position.TopRight;
+                                break;
+                            case "Top left corner":
+                                Position = Position.TopLeft;
+                                break;
+                            case "Bottom right corner":
+                                Position = Position.BottomRight;
+                                break;
+                            case "Bottom left corner":
+                                Position = Position.BottomLeft;
+                                break;
+                            case "Center":
+                                Position = Position.Center;
+                                break;
+                        }
+                    }
+                }
+                catch (Exception ex)
+                {
+                    Logger.LogError("Failed to read changed settings", ex);
+                }
+            }
+        }
+    }
+
+    private PowerAccentActivationKey _activationKey = PowerAccentActivationKey.Both;
+
+    public PowerAccentActivationKey ActivationKey
+    {
+        get
+        {
+            return _activationKey;
+        }
+
+        set
+        {
+            _activationKey = value;
+        }
+    }
+
+    private Position _position = Position.Top;
+
+    public Position Position
+    {
+        get
+        {
+            return _position;
+        }
+
+        set
+        {
+            _position = value;
+        }
+    }
+
+    private int _inputTime = 200;
+
+    public int InputTime
+    {
+        get
+        {
+            return _inputTime;
+        }
+
+        set
+        {
+            _inputTime = value;
+        }
+    }
+
+    public char[] GetLetterKey(LetterKey letter)
+    {
+        return GetDefaultLetterKey(letter);
+    }
+
+    public static char[] GetDefaultLetterKey(LetterKey letter)
+    {
+        switch (letter)
+        {
+            case LetterKey.A:
+                return new char[] { 'à', 'â', 'á', 'ä', 'ã', 'å' };
+            case LetterKey.C:
+                return new char[] { 'ć', 'ĉ', 'č', 'ċ', 'ç', 'ḉ' };
+            case LetterKey.E:
+                return new char[] { 'é', 'è', 'ê', 'ë', '€' };
```

Comment:
```suggestion
                return new char[] { 'é', 'è', 'ê', 'ë', 'ē', 'ē', '€' };
```

Reply:
What's the difference between these two? They look the same to me.

Reply:
my mistake